### DISCUSSION TITLE
Adopt World Geographical Scheme for Recording Plant Distributions (WGSRPD)?

### DISCUSSION THREAD

**Discussion by mkoo** [ArctosDB/arctos#8048] [General]
I have a question for the community:
How many people are aware of and/ or use the **World Geographical Scheme for Recording Plant Distributions (WGSRPD)**

https://www.tdwg.org/standards/wgsrpd/
[Full report here](https://github.com/tdwg/wgsrpd/blob/master/109-488-1-ED/2nd%20Edition/TDWG_geo2.pdf)
It doesnt appear to be updated currently with the last 2nd edition released in 2006 but maybe that's not a problem and we just use Level 1-2 or whatever makes sense.

I'm going to check out the spatial data but this may help with #6930 #5595 etc

**Comment by dustymc** [comment]
In general I don't think there are any problems with multiple Standards, as long as the higher geography isn't the same. Having two "over there" isn't tenable for lots of reasons, but having "over there, stopping at this line" and "over there, stopping at that line" (eg two spatial shapes and two distinct higher_geog concatenations referring to one PLACE) doesn't seem in any way problematic. (One perfect everything source of course would be simpler, if that's hiding in there somewhere...)

https://github.com/ArctosDB/arctos/issues/5595 seems like a "simple" infrastructure issue to me - I've got the shape, I'm just having trouble finding the cpu-or-something to DO STUFF with it. I think it's just another symptom of https://github.com/ArctosDB/arctos/issues/6521, I really need some help resolving this.

https://github.com/ArctosDB/arctos/issues/6930 is definitely blocked by https://github.com/ArctosDB/arctos/issues/5595, I can't load shapes at all at the moment, HELP!
